### PR TITLE
Treegrid example: Update to use APG Example template for documentation

### PR DESCRIPTION
**Pull request by jongund**

### CODE REVIEW

File: examples/treegrid/treegrid-1.html [hunk 1]
```diff
@@ -83,18 +77,8 @@ <h1>Treegrid Email Inbox Example</h1>
     The following example demonstrates how the
     <a href="../../#treegrid">treegrid design pattern (TBD)</a>
     can be used to make an interactive tree that enables users to navigate the hierarchical structure of email conversations
-    and also provides the ability to navigate to elements that describe each email, such as subject and sender.
-  </p>
-  <p>
-    This same example can be used in three ways.
-  </p>
-  <ul>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: Not sure we want to eliminate these operating modes; topic of discussion. We could make them three different example pages, but I am not sure that is the best approach in this case.

File: examples/treegrid/treegrid-1.html [hunk 2]
```diff
@@ -166,44 +150,24 @@ <h2 id="ex_label">Example</h2>
...
-      <li>aria-labelledby or aria-describedby for headers? Not currently used, awaiting discussion. Some UA + screen reader combinations seem to be determining this automatically via the table structure.</li>
-      <li>aria-readonly: in ARIA 1.0, a grid/treegrid is editable by default. However, there is no default in ARIA 1.1. Firefox currently implements the ARIA 1.0 concept for this, which means that "editable" is read for every cell unless aria-readonly="true" is used on the treegrid. <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1359482">A bug has been filed in Mozilla to not propagate the readonly/editable state to descendant gridcells unless it is specified via aria-readonly</a>.</li>
+      <li>This example uses the roving tabindex technqiue to manage the tabbing into and our of the treegrid and tabindex values are managed through javascript.</li>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: Change "technqiue " to "technique". There are lots more spelling errors. Please check for spelling.
Some examples include: adddress, thetreegrid, girdcell, indictes, avaiable, aria-poinset

File: examples/treegrid/treegrid-1.html [hunk 3]
```diff
@@ -403,11 +537,11 @@ <h2>Javascript and CSS Source Code</h2>
...
-        <a href="css/treegrid-1.css" type="tex/css">example_name.css</a>
+        <a href="css/treegrid-1.css" type="text/css">treegrid-1.css</a>
       </li>
       <li>
         Javascript:
-        <a href="js/treegrid-1.js" type="text/javascript">example_name.js</a>
+        <a href="js/treegrid-1.js" type="text/javascript">treegrid-1.js</a>
       </li>
     </ul>
   </section>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: There's a link at the end of the file back to the pattern than needs to be updated. Github doesn't let me comment on that line.

File: examples/treegrid/treegrid-1.html [hunk 4]
```diff
@@ -83,18 +77,8 @@ <h1>Treegrid Email Inbox Example</h1>
     The following example demonstrates how the
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: On line 36 of [Example-Template.html](https://github.com/w3c/aria-practices/blob/master/examples/coding-template/Example-Template.html) there is a set of four links in a nav region that are missing from this page.

**Comment by jongund** [comment]
Matt,

Thank you for the detailed feedback, I think I fixed most of the issues.

Can talk about how to deal with initial focus options on the call today.

Jon

File: examples/treegrid/treegrid-1.html [hunk 5]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>false</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: Spelling error: assisitve

File: examples/treegrid/treegrid-1.html [hunk 6]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-hidden=<q>true</q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
+              <li><code>aria-hidden</code> is used to control the information available to assisitve technologies.</li>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: Spelling error: assisitve

File: examples/treegrid/treegrid-1.html [hunk 7]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-level=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            <ul>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: For simplicity and clarity, I recommend replacing the bullets in this list with the following to be consistent with the wording we use in the tree examples:

* Defines the level of the row in the hierarchical treegrid structure.
* Counting is one-based.
* Root rows have aria-level=“1”.

File: examples/treegrid/treegrid-1.html [hunk 8]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-setsize=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            Identifies the number of message rows that responded to the parent e-mail message.
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: Recommend the following wording to be consistent with the tree examples:

Defines the number of rows in the set of rows that are in the same branch and at the same level within the hierarchy.

File: examples/treegrid/treegrid-1.html [hunk 9]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+          </td>
+        </tr>
+        <tr>
+          <td>
+          </td>
+          <th scope="row"><code>aria-posinset=<q><em>number</em></q></code></th>
+          <td><code>tr</code></td>
+          <td>
+            Identifies the index of message row in the current set of message rows that responded to the parent e-mail message.
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: I recommend the following wording to be consistent with the tree examples:

* Defines the position of the row within the set of other rows that are in the same branch and at the same level within the hierarchy.
* Counting is one-based, not zero-based.

File: examples/treegrid/treegrid-1.html [hunk 10]
```diff
@@ -370,30 +330,200 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
...
+          </td>
+        </tr>
+        <tr>
+          <td></td>
+          <th scope="row"><code>tabindex=<q>-1</q></code></th>
+          <td><code>td</code></td>
+          <td>
+            <ul>
+              <li>Makes the <code>gridcell</code> element focusable without including it in the tab sequence of the page.</li>
+              <li>All <code>row</code> and <code>girdcell</code> elements are focusable, but only one is included in the tab sequence.</li>
```
Comment (mcking65) [inline examples/treegrid/treegrid-1.html]: spelling error: girdcell

**Comment by jongund** [comment]
Added back in the deleted content

**Comment by jongund** [comment]
Matt,

I think I have fixed all the problems you have identified

**Pull request merged**